### PR TITLE
Refactor the date components

### PR DESCRIPTION
**Pull request by Windvis**
This modernizes the date components:
- remove usage of webuniversum components
- remove usage of moment

### CODE REVIEW

**Comment by Windvis** [comment]
Works as expected in loket. It only has some styling conflicts because of the generic table styles in ember-vo-datatable but that will be removed anyways.

File: addon/utils/date.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export function formatDate(date) {
+  if (!(date instanceof Date) || isNaN(date)) {
+    return '';
+  }
+
+  let day = `${date.getDate()}`.padStart(2, '0');
```
Comment (aatauil) [inline addon/utils/date.js:6]: Great find on the padStart function! I didnt know that existed, would have been handy on other components like the timepicker that I created in appuniversum :p

**Comment by aatauil** [comment]
Other than the style conflict in loket that you mentioned above, I did not find any other issues and everything seems to work as expected :)

**Comment by Windvis** [comment]
That can be fixed by commenting out `@import 'ember-vo-webuniversum-data-table'; // Deprecated` in the app.scss file, but I didn't want to spend time to find out if it affected other things so I didn't remove that file in this PR. Once all webuniversum components and styles are removed we can do a sweep over the app and find out where things look broken.

**Pull request merged**